### ISSUE TITLE
Invisible Wall on Luiz's Apartment entrance

### ISSUE THREAD

**Issue by RealAlexMouse** [valentyn-l/GTAIV.EFLC.Various.Fixes#49]
![Screenshot 2024-08-11 200447](https://github.com/user-attachments/assets/a347f13a-f479-425e-8ce7-9a518563ea9c)
When I try to enter Luiz's Algonquin apartment in TBoGT, There appears to be some sort of invisible wall or barrier blocking me from entering. I can punch and shoot it and it creates marks on the barrier. I have no idea what's causing this and I was hoping you could look into it.

Thank you

**Comment by valentyn-l** [comment]
Is this an issue in vanilla TBoGT? Have you tried the same area without any mods?

**Comment by RealAlexMouse** [comment]
> Is this an issue in vanilla TBoGT? Have you tried the same area without any mods?

The only other mod I have installed besides this one is "Fusion Fix". Other than that, this is vanilla.

**Comment by Attramet** [comment]
> same here, steam version, fusionfix + various fixes

is it still happening with 2.1 update? because I can't reproduce it

**Comment by srhiers** [comment]
yes I tried it yesterday, I played the game 1 year ago an the bug was back then too

**Comment by Attramet** [comment]
> yes I tried it yesterday, I played the game 1 year ago an the bug was back then too

I guess I found the culprit, try to remove manhat05_7.wbn file from VariousFixes.img and check if issue is gone

**Comment by srhiers** [comment]
fixed, thank you
